src/components/street-segment.js

```diff
@@ -252,6 +252,19 @@ AFRAME.registerComponent('street-segment', {
         });
       });
     }
+
+    if (componentsToGenerate?.striping?.length > 0) {
+      componentsToGenerate.striping.forEach((stripe, index) => {
+        this.el.setAttribute(`street-generated-striping__${index}`, {
+          striping: stripe.striping,
+          segmentWidth: this.data.width,
+          length: this.data.length,
+          positionY: stripe.positionY || 0.05, // Default to 0.05 if not specified
```

Comment:
You'll have a bug if you want positionY 0 here. For default value use the `??` operator instead of `||`.

Reply:
thanks